src/common/components/theme.tsx

```diff
@@ -33,14 +34,13 @@ export class ThemeInner extends React.Component<ThemeInnerProps> {
     }
 
     public render(): JSX.Element {
-        const enableHighContrast = this.isHighContrastEnabled(this.props);
-        const className = css('theme-switcher', enableHighContrast && 'high-contrast-theme');
+        const classNames = ['theme-switcher'];
```

Comment:
I'm not really convinced that maintaining this className is necessary, but I thought removing it would be high enough risk of us finding some "exciting" reason to revert it that I'd rather keep removing it to a separate PR